### ISSUE TITLE
Toggling window option.

### ISSUE THREAD

**Issue by qqkookie** [OpenRCT2/OpenRCT2#7876]
**OS:** Windows 10
**Version:** 0.2.0
**Commit/Build:** Vanilla version

Most windows opened by clicking toolbar button or ride track stay open. For example, Map, Park info,  Staff iList, Guest List, Finance, new ride/attraction construction, Ride status info, individual guest or staff info, park rating, guest numbers, etc. Clicking toolbar button again will bring the open window front and the window stays open.  

IMHO, these windows should be closed  on second click. i.e. they should toggle on successive click on the toolbar button .  Such behavior can be controlled in Option config like below:

 
![togg](https://user-images.githubusercontent.com/39845532/43994804-d36df388-9ddd-11e8-8d03-c84922344dfd.png)

--------------------------

- [X] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Comment by ocalhoun6** [comment]
It would be nice if this also worked when you open one of those windows via a shortcut key.

For example, I have the window for cutaway view assigned to the shortcut key '='. It would be nice if I could just hit = again to close the window.

**Comment by qqkookie** [comment]
Yes, toggling  window  can work with shortcut key as well as Toolbar button. Press once window will pop up. Press same key again will close the window.
